### PR TITLE
Fix on Extended math extension

### PR DESCRIPTION
**Pull request by Bouh**
Fix review from https://github.com/4ian/GDevelop-extensions/pull/57

### CODE REVIEW

**Comment by Add00** [comment]
I took 4ians advice and placed each constant in its own expression.

[ExtendedMathSupport.zip](https://github.com/4ian/GDevelop-extensions/files/5397603/ExtendedMathSupport.zip)

**Comment by 4ian** [comment]
Thanks! Should we rename each expression so that it is in PascaleCase (`Pi`, etc...)?
For consistency with what we do for GDevelop expressions.

**Comment by Add00** [comment]
I'm like 99% sure that everything is in PacaleCase now

[ExtendedMathSupport.zip](https://github.com/4ian/GDevelop-extensions/files/5397760/ExtendedMathSupport.zip)

**Comment by Bouh** [comment]
Your zip don't use the lastest content from this PR.

File: Extensions/ExtendedMath.json [hunk 1]
```diff
@@ -531,7 +539,7 @@
       "description": "Returns half Pi.",
       "fullName": "Half Pi",
       "functionType": "Expression",
-      "name": "haflPi",
+      "name": "HaflPi",
```
Comment (4ian) [inline Extensions/ExtendedMath.json:542]: HalfPi

**Review by 4ian — APPROVED** [review]

File: Extensions/ExtendedMath.json [hunk 2]
```diff
@@ -547,6 +477,234 @@
...
+        }
+      ],
+      "parameters": [],
+      "objectGroups": []
+    },
+    {
+      "description": "Returns half Pi.",
+      "fullName": "Half Pi",
+      "functionType": "Expression",
+      "name": "HaflPi",
```
Comment (Bouh) [inline Extensions/ExtendedMath.json:542]:
```suggestion
      "name": "HalfPi",
```

**Comment by Add00** [comment]
Oh your right I forgot to swap by branch from master to fix extended math. I fixed some minor spelling errors.

[ExtendedMathSupport.zip](https://github.com/4ian/GDevelop-extensions/files/5397810/ExtendedMathSupport.zip)

**Comment by Bouh** [comment]
Thank you everyone, the version in this PR is same that's yours.
I merge the fix, all will be available directly in the extension list on GDevelop!
No download required ;)

**Pull request merged**